app/controllers/admin/orders_controller.rb

```diff
@@ -6,4 +6,11 @@ def update
     redirect_to "/admin"
   end
 
+  def show
+    if current_user == nil || current_user.role != "admin"
```

Comment:
User shouldnt be able to access page unless they are an admin.

Reply:
boom!

Reply:
![](https://media3.giphy.com/media/8abAbOrQ9rvLG/giphy.gif)

Reply:
bing bango